util/mkdef.pl

```diff
@@ -386,7 +386,10 @@ sub writer_VMS {
 _____
 
     if (defined $version) {
-        my ($libvmajor, $libvminor) = $version =~ /^(\d+)_(\d+)$/;
+        print STDERR "DEBUG: \$version = $version\n";
```

Comment:
Is this a leftover or intended to go in?

Reply:
Oops.
Leftover. Will fix